pkg/commands/compute/language_assemblyscript.go

```diff
@@ -3,126 +3,105 @@ package compute
 import (
 	"fmt"
 	"io"
-	"os"
 	"path/filepath"
-	"time"
 
 	fsterr "github.com/fastly/cli/pkg/errors"
-	fstexec "github.com/fastly/cli/pkg/exec"
-	"github.com/fastly/cli/pkg/filesystem"
 	"github.com/fastly/cli/pkg/manifest"
 	"github.com/fastly/cli/pkg/text"
 )
 
-// ASSourceDirectory represents the source code directory.
-const ASSourceDirectory = "assembly"
+// AsCompilation is a language specific compilation target that converts the
+// language code into a Wasm binary.
+const AsCompilation = "asc"
```

Comment:
Is this still needed? I would have thought that the CLI no longer needs to worry about the tool being used for compilation.